src/js/popup.js

```diff
@@ -398,6 +420,11 @@ function refreshPopup() {
     }
   }
 
+  // if there is any saved error text, fill the error input with it
```

Comment:
Should this part be combined with "show error layout if the user was writing an error report" (and live in `init()`)?

Reply:
I tried that to start with, but it caused a strange bug as if it created an infinite loop when opening the popup.